src/components/task.vue

```diff
@@ -455,6 +455,7 @@ export default {
 
       const elementDestinationUrl = elementDestination.value;
       if (elementDestinationUrl) {
+        sessionStorage.setItem('sessionUrlActionBlocker', document.referrer);
```

Comment:
Similar as the code below consider to add some comment to document the meaning of `sessionUrlActionBlocker`
```
// Save the referring URL to sessionStorage for future verification
```